go.mod

```diff
@@ -61,3 +64,5 @@ require (
 )
 
 replace golang.org/x/net => golang.org/x/net v0.0.0-20210119194325-5f4716e94777
+
+replace github.com/iver-wharf/wharf-api-client-go/v2 => ../wharf-api-client-go
```

Comment:
Waiting for the following PRs to be merged before I can remove this:

- https://github.com/iver-wharf/wharf-api-client-go/pull/37
- https://github.com/iver-wharf/wharf-api-client-go/pull/38